docs/gallery-submission.md

```diff
@@ -49,12 +49,11 @@ Join our [Tableau Developer Program](https://www.tableau.com/developer) to gain
 
 ### Step 2: Build your integration
 
-After you've signed up for our developer program, use the information in this Connector SDK documentation to get started building a connector.
```

Comment:
Can you add a link to the bug artifacts page (https://tableau.github.io/connector-plugin-sdk/docs/bug-artifacts) here as well? We're trying to advertise that page a bit more to try and reduce the amount of times we need to respond asking for more artifacts. Thanks!